provision/acc_provision/templates/aci-containers.yaml

```diff
@@ -390,10 +390,13 @@ data:
     {
         "flavor": {{config.flavor|json}},
         "log-level": {{config.logging.controller_log_level|json}},
+        {% if config.aci_config.capic or config.kube_config.opflex_mode != "overlay" %}
```

Comment:
Might be better to use config.flavor != "k8s-overlay".

Reply:
OK I like that suggestion. I'll update the PR shortly

Reply:
I've updated all checks to use flavor instead of opflex_mode. Let me know if you have any other comments/suggestions